### ISSUE TITLE
Can you provide the complete training code?

### ISSUE THREAD

**Issue by azsxza** [a-nagrani/SVHF-Net#2]
I used Keras to implement the network in the paper, but the network did not learn at all, the loss did not decline, and the accuracy was always about 0.5.
